docs/src/main/asciidoc/writing-native-applications-tips.adoc

```diff
@@ -99,17 +99,17 @@ The final order of business is to make the configuration file known to the `nati
 ----
 quarkus.native.additional-build-args =\
     -H:+UnlockExperimentalVMOptions,\
-    -H:ResourceConfigurationFiles=resources-config.json,\
+    -H:ResourceConfigurationFiles=resource-config.json,\
```

Comment:
I will let @zakkak chime in but do we even need to register the file if we are following the convention?